### ISSUE TITLE
[1.3] Error: Surprising path: /packages/modules.js Cant run updated/new 1.3 projects

### ISSUE THREAD

**Issue by briarfox** [meteor/meteor#6619]
Windows 10 x64
Will throw this error on a updated projects and newly created projects.

```
=> Started proxy.
=> Started MongoDB.

C:\Users\Chris\AppData\Local\.meteor\packages\templating\1.1.7\plugin.compileTemplatesBatch.os\npm\node_modules\meteor\promise\node_modules\meteor\promise\node_modules\meteor-promise\promise_server.js:116
      throw error;
            ^
Error: Surprising path: /packages/modules.js
    at toDosPath (C:\Users\Chris\AppData\Local\.meteor\packages\meteor-tool\1.3.0_3\mt-os.windows.x86_32\tools\static-assets\server\mini-files.js:34:13)
    at C:\Users\Chris\AppData\Local\.meteor\packages\meteor-tool\1.3.0_3\mt-os.windows.x86_32\tools\static-assets\server\mini-files.js:82:16
    at Array.map (native)
    at Function._.map._.collect (C:\Users\Chris\AppData\Local\.meteor\packages\meteor-tool\1.3.0_3\mt-os.windows.x86_32\dev_bundle\lib\node_modules\underscore\underscore.js:97:56)
    at C:\Users\Chris\AppData\Local\.meteor\packages\meteor-tool\1.3.0_3\mt-os.windows.x86_32\tools\static-assets\server\mini-files.js:79:16
    at ImportScanner._ensureSourcePath (C:\tools\isobuild\import-scanner.js:260:25)
    at C:\tools\isobuild\import-scanner.js:120:28
    at Array.forEach (native)
    at ImportScanner.addInputFiles (C:\tools\isobuild\import-scanner.js:119:11)
    at C:\tools\isobuild\compiler-plugin.js:756:15
    at Array.forEach (native)
    at Function.computeJsOutputFilesMap (C:\tools\isobuild\compiler-plugin.js:726:19)
    at ClientTarget._emitResources (C:\tools\isobuild\bundler.js:910:8)
    at C:\tools\isobuild\bundler.js:677:12
    at C:\tools\utils\buildmessage.js:359:18
    at [object Object].withValue (C:\tools\utils\fiber-helpers.js:89:14)
    at C:\tools\utils\buildmessage.js:352:34
    at [object Object].withValue (C:\tools\utils\fiber-helpers.js:89:14)
    at C:\tools\utils\buildmessage.js:350:23
    at [object Object].withValue (C:\tools\utils\fiber-helpers.js:89:14)
    at Object.enterJob (C:\tools\utils\buildmessage.js:324:26)
    at ClientTarget.make (C:\tools\isobuild\bundler.js:668:18)
    at C:\tools\isobuild\bundler.js:2509:14
    at C:\tools\isobuild\bundler.js:2599:20
    at Array.forEach (native)
    at Function._.each._.forEach (C:\Users\Chris\AppData\Local\.meteor\packages\meteor-tool\1.3.0_3\mt-os.windows.x86_32\dev_bundle\lib\node_modules\underscore\underscore.js:79:11)
    at C:\tools\isobuild\bundler.js:2598:7
    at C:\tools\utils\buildmessage.js:271:13
    at [object Object].withValue (C:\tools\utils\fiber-helpers.js:89:14)
    at C:\tools\utils\buildmessage.js:264:29
    at [object Object].withValue (C:\tools\utils\fiber-helpers.js:89:14)
    at C:\tools\utils\buildmessage.js:262:18
    at [object Object].withValue (C:\tools\utils\fiber-helpers.js:89:14)
    at C:\tools\utils\buildmessage.js:253:23
    at [object Object].withValue (C:\tools\utils\fiber-helpers.js:89:14)
    at Object.capture (C:\tools\utils\buildmessage.js:252:19)
    at Object.exports.bundle (C:\tools\isobuild\bundler.js:2490:31)
    at C:\tools\runners\run-app.js:591:36
    at Function.run (C:\tools\tool-env\profile.js:489:12)
    at bundleApp (C:\tools\runners\run-app.js:581:34)
    at AppRunner._runOnce (C:\tools\runners\run-app.js:634:35)
    at AppRunner._fiber (C:\tools\runners\run-app.js:887:28)
    at C:\tools\runners\run-app.js:411:12
```

**Comment by unmonk** [comment]
Also receiving this on a fresh meteor install.  Windows 10x64

=> Started proxy.
=> Started MongoDB.
=> Errors prevented startup:

   While loading plugin `compileTemplatesBatch` from package `templating`:
   module.js:338:15: Cannot find module '../modules/es6.object.get-own-property-descriptor'
   at Function.Module._resolveFilename (module.js:338:15)
   at Function.Module._load (module.js:280:25)
   at Module.require (module.js:364:17)
   at require (module.js:380:17)
   at Object.<anonymous>

**Comment by benjamn** [comment]
This should be fixed if you do `meteor update --release 1.3.1`.
